java/pom.xml

```diff
@@ -127,7 +127,12 @@
         Managed from a conflict between 1.6.0 and 3.0.0 within Spark, and 1.17.2 in Trino.
         -->
         <okio.version>3.12.0</okio.version>
-        <!-- Trino testing declares an old version. Managed from 1.2.3. -->
+        <!--
+        Spark declares an old version of RoaringBitmap with vulnerabilities. We have upgraded the version
+        ahead of the Spark version to address detected vulnerabilities.
+        -->
+        <roaringbitmap.version>1.3.0</roaringbitmap.version>
+         <!-- Trino testing declares an old version. Managed from 1.2.3. -->
```

Comment:
It looks like an extra space has been added at the start of this line, so it doesn't match the indentation of the file any more. This is really minor though.